training/MESSAGING_OVERVIEW.md

```diff
@@ -57,21 +57,19 @@ The three components of the `registerFunction` method are the `cmd` string, the
 
 It is important the Chapel function is formatted correctly or the server will not compile. The expected formatting for a `serverFunctionMsg` is:
 ```chapel
-proc serverFunctionMsg(cmd: string, payload: string, argSize: int, st: borrowed SymTab): MsgTuple throws {}
+proc serverFunctionMsg(cmd: string, msgArgs: borrowed MessageArgs, st: borrowed SymTab): MsgTuple throws {}
 ```
 
 The parameters in this function each server a specific purpose:
 - `cmd` is the command string used to access this function. It is used mostly for debugging as the `cmd` string used to access the function will be captured in the logger messages.
-- `payload` is the JSON string of arguments passed from the client.
-- `argSize` lets Chapel know how many arguments are in the payload JSON so it can properly create a map using the JSON.
+- `msgArgs` is the recieved information that the client passes as a `repMsg`. The information about the number of arguements ia bundled and will be pared accordingly with the denoted passed types. Individual arguements will need to be extracted on the server side.

```

Comment:
`repMsg` is not entirely correct here. I would state that it contains parameters required for the message processing instead.